portfolio/src/test/java/com/google/sps/HashTableTest.java

```diff
@@ -43,9 +43,9 @@ public void oneBlogPostForEachTag() {
     //Return a Hashmap with LinkedList of size one for each tag.
     BlogHashMap actual = new BlogHashMap();
     List<BlogMessage> temp = new ArrayList<>();
-    temp.add(new BlogMessage((long) 1, "#general", "I enjoy pop music", "", "Steven", new ArrayList<String>(), System.currentTimeMillis()));
-    temp.add(new BlogMessage((long) 2, "#wellbeing", "Country music is the best music", "", "Tayyaba", new ArrayList<String>(), System.currentTimeMillis()));
-    temp.add(new BlogMessage((long) 3, "#music", "Classical music makes you smarter", "", "Andrew", new ArrayList<String>(), System.currentTimeMillis()));
+    temp.add(new BlogMessage((long) 1, "#general", "I enjoy pop music", "", "Steven", "", new ArrayList<String>(), System.currentTimeMillis()));
```

Comment:
"temp" in general is really not a good name for variables. Consider using "messages" or "blogMessages"